synapse/rest/media/v1/media_storage.py

```diff
@@ -127,18 +133,26 @@ async def finish():
                     f.flush()
                     f.close()
 
+                    spam = await self.spam_checker.check_media_file_for_spam(
+                        ReadableFileWrapper(self.clock, fname), file_info
+                    )
+                    if spam:
+                        logger.info("Blocking media due to spam checker")
```

Comment:
Should we include some info about what was blocked while logging?

Reply:
We don't actually know very much about the media annoyingly, e.g. we don't know the media ID. I hope we can extract enough info from the request logs.

Reply:
Doesn't `file_info` have some stuff in it?

Reply:
Yeah, it has some stuff, but not the media ID. We could log if its a remote download or not, but I'm not sure that is particularly useful given you can get that from the request logs?

Reply:
Oh, I see all that info is pretty much in the requests logs. 👍